lib/rucio/client/uploadclient.py

```diff
@@ -289,9 +300,9 @@ def upload(self, items, summary_file_path=None, traces_copy_out=None):
                 json.dump(final_summary, summary_file, sort_keys=True, indent=1)
 
         if num_succeeded == 0:
-            raise NoFilesUploaded()
+            raise NoFilesUploaded(str(collections.Counter(summary_exception)))
```

Comment:
Could we omit these changes for now and think again about how we pass these error information back?

Reply:
Well. I did already two versions. We greed on previous one, we agreed on this one. I am not going to apply another already third one.

Reply:
I didn't agree ^^ Back then I started doing my own version based on your commit but I wasn't able to finish it and now I also don't have time to finish it.